styles/index.css

```
@@ -0,0 +1,281 @@
+:root {
+  /* Primary color */
+  --primary-100: #3692ff;
+  --primary-200: #1967d6;
+  --primary-300: #1251aa;
+
+  /* Secondary color */
+  --secondary-50: #f9fafb;
+  --secondary-100: #f3f4f6;
+  --secondary-200: #e5e7eb;
+  --secondary-400: #9ca3af;
+  --secondary-500: #6b7280;
+  --secondary-600: #4b5563;
+  --secondary-700: #374151;
+  --secondary-800: #1f2937;
+  --secondary-900: #111827;
+
+  /* Error color */
+  --error-500: #f74747;
+
+  /* Disable color */
+  --disable-400: #9ca3af;
+
+  /* Font style */
+  --font-primary: 'Pretendard Variable', sans-serif;
+  --font-size-base: 16px;
+}
+
+body {
+  min-width: 350px;
+  overflow-x: hidden;
+}
+
+/* 정보 섹션 반응형(테블릿) */
+@media (max-width: 1010px) {
+  .info {
+    margin: 24px;
+    gap: 52px;
+  }
+
+  .info__container {
+    height: 750px;
+    gap: 52px;
+  }
+
+  .info__wrapper {
+    width: 696px;
+    height: 750px;
+    flex-direction: column;
+    gap: 24px;
+  }
+
+  .info__title br {
+    display: none;
+  }
+
+  .info__image {
+    width: 696px;
+    height: 524px;
+  }
+
+  .info__content {
+    align-self: flex-start;
+    padding-left: 24px;
+  }
+
+  .info__wrapper--reverse .info__content {
+    align-self: flex-end;
+    padding-right: 24px;
+  }
+}
+
+/* 정보 섹션 반응형(모바일) */
+@media (max-width: 725px) {
+  .info {
+    margin: 52px 15px;
+    gap: 40px;
+  }
+
+  .info__container {
+    height: 440px;
+    gap: 40px;
+  }
+
+  .info__wrapper {
+    width: 344px;
+    height: 440px;
+    gap: 40px;
+  }
+
+  .info__content {
+    gap: 8px;
+  }
+
+  .info__image {
+    width: 344px;
+    height: 259px;
+  }
+
+  .info__category {
+    font-size: 16px !important;
+  }
+
+  .info__title {
+    font-size: 24px !important;
+    line-height: 32px !important;
+    margin-bottom: 8px;
+  }
+
+  .info__description {
+    font-size: 16px !important;
+    line-height: 26px !important;
```

사실 Figma에 컬러파레트와 폰트 속성이 표로 나와있길래 `typography.css` 파일로 스타일을 정의한 뒤, 클래스로 불러서 폰트 속성을 조절하게 구현했다보니 `!important` 키워드를 많이 사용했습니다. 
`important`키워드를 자주 사용해도 괜찮은지 리뷰 부탁드립니다!

---

!important 사용은 지양하는 것이 좋습니다.

현재 typegraphy 관련 파일이 다른 CSS 파일보다 먼저 import되고 있기 때문에, 같은 명시도를 가진 스타일이라면 !important 없이도 index.css가 나중에 로드되면서 .info__description 스타일이 적용될 것입니다.
하지만 import 순서에 의존하는 방식은 유지보수 시 신경 써야 할 부분이 많아 추천드리지 않습니다.

대신, 우선순위를 높이는 방식으로 CSS를 작성하는 것을 추천드립니다.

```suggestion
  .info  .info__category {
    font-size: 16px;
  }

  .info  .info__title {
    font-size: 24px;
    line-height: 32px;
    margin-bottom: 8px;
  }

  .info  .info__description {
    font-size: 16px;
    line-height: 26px;
```

[mdn important 관련](https://developer.mozilla.org/ko/docs/Web/CSS/CSS_cascade/Specificity#!important_%EC%98%88%EC%99%B8) 링크입니다. 왜 해당 패턴을 지양해야하는지에 대해 자세히 설명되어 있으니 읽어보세요~

---

명시도로 해결할 수 있는 문제였는데 구현에 급급하다보니 !important를 남용한 것 같네요..! 2주차엔 수정한 뒤 pr 올리도록 하겠습니다! 감사합니다!